bin/rule_parser.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import ast
+import yaml
+
+LOGIC_REPLACE = {'||': 'or',
+                 '&&': 'and',
+                 '!=': 'is not',
```

Comment:
This does not need to be translated in Python and 'is not' is probably not what you want as 'is' does an identity check.